sdks/go/pkg/beam/pardo_test.go

```diff
@@ -61,3 +71,65 @@ func TestFormatParDoError(t *testing.T) {
 		t.Errorf("formatParDoError(testFunction,2,1) = %v, want = %v", got, want)
 	}
 }
+
+func TestAnnotations(t *testing.T) {
+	m := make(map[string][]byte)
+	m["privacy_property"] = []byte("differential_privacy")
+	doFn := &AnnotationsFn{Annotations: m}
+
+	p := NewPipeline()
+	s := p.Root()
+
+	values := [2]int{0, 1}
+	col := CreateList(s, values)
```

Comment:
Consider using an Impulse instead, which simplifies the pipe to just the 2 transforms. The element would then be input `_ []byte` since impulse is traditionally ignored.)

However, I'm not going to worry about it at this juncture, since it would not meaningfully shrink the PR.